src/watertap_contrib/seto/unit_models/surrogate/med_tvc_surrogate.py

```diff
@@ -0,0 +1,1394 @@
+
+        sf = iscale.get_scaling_factor(
+            self.cooling_out_props[0].conc_mass_phase_comp["Liq", "TDS"]
+        )
+        iscale.constraint_scaling_transform(self.eq_cooling_salinity, sf)
+
+        sf = iscale.get_scaling_factor(self.brine_props[0].temperature)
+        iscale.constraint_scaling_transform(self.eq_brine_temp, sf)
+
+        sf = iscale.get_scaling_factor(
+            self.brine_props[0].conc_mass_phase_comp["Liq", "TDS"]
+        )
+        iscale.constraint_scaling_transform(self.eq_brine_salinity, sf)
+
+        sf = iscale.get_scaling_factor(self.feed_props[0].pressure)
+        iscale.constraint_scaling_transform(self.eq_feed_to_distillate_isobaric, sf)
+        iscale.constraint_scaling_transform(self.eq_feed_to_brine_isobaric, sf)
+        iscale.constraint_scaling_transform(self.eq_feed_to_cooling_isobaric, sf)
+
+        sf = iscale.get_scaling_factor(self.feed_props[0].flow_vol_phase["Liq"])
+        iscale.constraint_scaling_transform(self.eq_dist_vol_flow, sf)
+
+        sf = iscale.get_scaling_factor(self.brine_props[0].flow_vol_phase["Liq"])
+        iscale.constraint_scaling_transform(self.eq_brine_vol_flow, sf)
+
+        sf = iscale.get_scaling_factor(self.gain_output_ratio)
+        iscale.constraint_scaling_transform(self.eq_gain_output_ratio, sf)
+
+        sf = iscale.get_scaling_factor(self.specific_area)
+        iscale.constraint_scaling_transform(self.eq_specific_area, sf)
+
+        sf = iscale.get_scaling_factor(
+            self.heating_steam_props[0].flow_mass_phase_comp["Vap", "H2O"]
+        )
+        iscale.constraint_scaling_transform(self.eq_heating_steam_mass_flow_rate, sf)
+
+        sf = iscale.get_scaling_factor(
+            self.motive_steam_props[0].flow_mass_phase_comp["Vap", "H2O"]
+        )
+        iscale.constraint_scaling_transform(self.eq_motive_steam_mass_flow_rate, sf)
+
+        sf = iscale.get_scaling_factor(self.specific_energy_consumption_thermal)
+        iscale.constraint_scaling_transform(
+            self.eq_specific_energy_consumption_thermal, sf
+        )
+
+        sf = iscale.get_scaling_factor(self.thermal_power_requirement)
+        iscale.constraint_scaling_transform(self.eq_thermal_power_requirement, sf)
+
+        sf = (
+            iscale.get_scaling_factor(self.feed_cool_mass_flow)
+            * iscale.get_scaling_factor(
+                self.cooling_out_props[0].enth_mass_phase["Liq"]
+            )
+            * 1e3
+        )
+        iscale.constraint_scaling_transform(self.eq_feed_cool_mass_flow, sf)
+
+        sf = iscale.get_scaling_factor(self.feed_cool_vol_flow)
+        iscale.constraint_scaling_transform(self.eq_feed_cool_vol_flow, sf)
+
+        sf = (
+            iscale.get_scaling_factor(self.cooling_out_props[0].flow_vol_phase["Liq"])
+            / 3600
+        )
+        iscale.constraint_scaling_transform(self.eq_cool_vol_flow, sf)
+
+    def _get_gain_output_ratio_coeffs(self):
```

Comment:
Technically, there should also be `_get_stream_table_contents()` and `_get_performance_contents()`. Later, after successfully building, scaling, initializing and solving the model, you can report results with something like `m.fs.unit.report()`.

Reply:
You can check out unit models on WaterTAP to compare with.

Reply:
These 2 methods added.